src/vstest.console/Internal/ProgressIndicator/NoOpProgressIndicator.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine.Internal
+{
+    internal class NoOpProgressIndicator : IProgressIndicator
```

Comment:
In case we are removing NoOpProgressIndicator, we can remove interface as well.

Reply:
Need it for testing.